types/portal/additional_payload.go

```diff
@@ -31,3 +34,20 @@ func EncodeAdditionalPayload(index int64, destinationToken []byte) (bz []byte) {
 
 	return
 }
+
+func DecodeAdditionalPayload(bz []byte) (index int64, destinationToken []byte, err error) {
```

Comment:
Could we get a docstring added here? I think I included one when we were pair-coding on this